app/models/edition/workflow.rb

```diff
@@ -43,7 +43,7 @@ def finalised_states
         transitions from: %i[draft awaiting_review awaiting_factcheck scheduled], to: :published
       end
       event :schedule do
-        transitions from: %i[draft], to: :scheduled
+        transitions from: %i[draft awaiting_factcheck], to: :scheduled
```

Comment:
I didn't remove 'draft' here as that would break the flow for our users who don't have the feature flag